src/components/ListBooks.js

```diff
@@ -0,0 +1,22 @@
+export const books = [
+  {
+    tittle: 'Book1',
+    author: 'Author1',
+    completed: '64%',
+    chapter: '17',
+  },
+  {
+    tittle: 'Book2',
+    author: 'Author2',
+    completed: '8%',
+    chapter: '3: "A lesson learned"',
+  },
+  {
+    tittle: 'Book3',
+    author: 'Author3',
+    completed: '0%',
+    chapter: 'Introduction',
+  },
+];
```

Comment:
- [ ] [OPTIONAL] You can add unique `ids` to the books. To add unique IDs to the books in the books array, you could use a package called `uuid` library from `npm`. First, install the `uuid package` by running the following command in your project's directory:

```bash
npm install uuid
```

Then, update the books array by importing the v4 function from the uuid package and adding unique IDs to each book object:

```javascript
import { v4 as uuidv4 } from 'uuid';

export const books = [
  {
    id: uuidv4(),
    title: 'Book1',
    author: 'Author1',
    completed: '64%',
    chapter: '17',
  },
  {
    id: uuidv4(),
    title: 'Book2',
    author: 'Author2',
    completed: '8%',
    chapter: '3: "A lesson learned"',
  },
  {
    id: uuidv4(),
    title: 'Book3',
    author: 'Author3',
    completed: '0%',
    chapter: 'Introduction',
  },
];
```

Reply:
Thanks for your suggestion, I implemented it. 🙌